45.task/src/components/form.jsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import "./styles.css";
+import { Formik, Form, Field } from "formik";
+
+const FormComponent = () => { 
+
+    function checkName(value) {
+        if(!value) {
+            return 'Write Name';
+        }
+    }
+
+    function checkEmail(value) {
+        const regexp = /[\-_%+\.\w]+@([\w-]+\.)+[\w-]+/g
+        const checkValue = regexp.test(value) 
+        
+        if(!value) {
+            return 'Write Email';
+        } else if(!checkValue){
+            return 'Write correct Email';
+        }
+    }
+
+    function checkPhone(value) {
+        const regexp = /\d/;
+
+        if(!value) {
```

Comment:
Make messages more clear for users, because from you approach user don't understand what is wrong

```
if(!value) {
  return 'Field required, plz fill the value'
}

if (value.length < 12) {
  return 'Phone number must be 12 symbols';
}
        
if (!regexp.test(value)) {
 // if regexp is more complex, display the format of the number
 // for example - "Number format must be +38(XX) XX XX XXX"
  return 'Only numbers allowed';
}
```

Reply:
thx, for comments )